lib/jsonapi/plugs/content_type_negotiation.ex

```diff
@@ -33,8 +33,6 @@ defmodule JSONAPI.ContentTypeNegotiation do
     accepts =
       conn
       |> get_req_header("accept")
-      |> Enum.flat_map(&(String.split(&1, ",")))
-      |> Enum.map(&String.strip/1)
```

Comment:
removed these lines b/c we want to check to whole string to see if it contains the header we are looking for